tmt/steps/prepare/ansible.py

```diff
@@ -35,15 +37,15 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin):
     # Supported keys
     _keys = ["playbook", "extra-args"]
 
-    def __init__(self, step, data):
+    def __init__(self, step: str, data: Any) -> None:
```

Comment:
A `TypedDict` would be better, there are at least two keys know, `playbook` and `playbooks`.

Reply:
PTAL

Reply:
continue in other thread, this is not how it works